.cron.yml

```diff
@@ -9,7 +9,17 @@ jobs:
           type: decision-task
           treeherder-symbol: Nd
           target-tasks-method: nightly
-      when: [] # XXX Nightlies are temporarily disabled until Google Play catches up
+      when:
+          - {hour: 6, minute: 0}
```

Comment:
This LGTM, we're back to norma scheduling modulo the publishing part which is pending the GP saga.